.github/codecov.yml

```diff
@@ -0,0 +1,21 @@
+codecov:
+  require_ci_to_pass: yes
+  allow_coverage_offsets: True
```

Comment:
> Coverage Offsets
> Sometimes the pseudo-base chosen by Codecov will not be recent enough to directly substitute for the base commit of a pull-request. In this case, Codecov will try to use the diff between the pseudo-base and the true base to adjust the coverage information of the pseudo-base and account for recent changes.

Added this in case it helps improve coverage reporting for future PRs.